src/p_setup.c

```diff
@@ -1655,7 +1655,7 @@ static boolean P_LoadReject(int lumpnum, int totallines)
     // Otherwise, we need to allocate a buffer of the correct size
     // and pad it with appropriate data.
 
-    lumplen = W_LumpLengthWithName(lumpnum, "REJECT");
+    lumplen = W_LumpLength(lumpnum);
```

Comment:
@fabiangreffrath Why is `W_LumpLengthWithName` needed here?

[PUSS IV](https://www.doomworld.com/idgames/themes/xmas/12days-v4) has empty REJECT lumps with a size of 13 in the lump header table and crashes during the `memcpy` from the WAD in memory.

Reply:
Wait, [PUSS IV](https://www.doomworld.com/idgames/themes/xmas/12days-v4) has CWILV00 patch in the middle of MAP01:
<img width="498" height="322" alt="image" src="https://github.com/user-attachments/assets/f8bc1968-de35-4de2-a6d1-6833e6605049" />

It still works with DSDA-Doom and with unzipped WADs in Woof. Oh well.